src/devices/README.md

```diff
@@ -9,6 +9,7 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
 * [Bmp280](Bmp280/README.md)
 * [Dhtxx](Dhtxx/README.md)
 * [Hcsr04](Hcsr04/README.md)
+* [Lcm1602a1](Lcm1602a1/README.md)
```

Comment:
NIT: This should change both of the `Lcm1602a1` -> `CharacterLcd`

Reply:
Whoops, fixing.